datasafe-encryption/datasafe-encryption-impl/src/test/resources/config-test/null-test/expectation/mutable-null-keys.yaml

```diff
@@ -17,12 +17,14 @@ keys:
     algo: AES
     size: 256
   encrypting:
-    algo: RSA
-    size: 2048
-    sigAlgo: SHA256withRSA
+    algo: ECDH
+    size: 256
+    sigAlgo: SHA256withECDSA
+    curve: Curve25519
   signing:
-    algo: RSA
-    size: 2048
-    sigAlgo: SHA256withRSA
+    algo: ECDH
```

Comment:
Curve25519 was designed to be used with EdDSA endeavour using a curve compatible with ECDSA e.g (secp256r1).